app/meldingen/incident/definitions/wizard-step-vulaan.js

```diff
@@ -41,16 +42,78 @@ export default {
           cols: 6,
           label: 'Wat is de naam van de rederij? (niet verplicht)',
           subtitle: 'Als u begint met typen verschijnt vanzelf een lijst met rederijen',
-          values: [
-            'Geen',
-            'Lovers',
-            'Aemstelland',
-            'Staets'
-          ],
+          path: 'text_extra',
+          values: {
```

Comment:
can this be done as an import from a definition? see ../admin/definitions

Reply:
we can do this later after mvp